gef-ui/server/utils/multer-filter.utils.js

```diff
@@ -14,25 +13,11 @@ const formatBytes = (bytes, decimals = 2) => {
 };
 
 const multerFilter = (req, file, cb) => {
-  const { documentType } = req.params;
-
-  const manualInclusionFileSize = 12582912; // == 12mb
-  const defaultFileSize = 10485760; // == 10mb
-  const maxFileSize = documentType === 'manual-inclusion-questionnaire' ? manualInclusionFileSize : defaultFileSize;
-  const fileSize = parseInt(req.headers['content-length'], 10);
-
-  if (file.originalname.match(new RegExp(`\\.(${DEFAULT_ALLOWED_FORMATS.join('|')})$`))) {
-    // if the document type is manual inclusion, then the max file size is 12mb
-    if (fileSize <= maxFileSize) {
-      cb(null, true);
-    } else {
-      cb({
-        message: `${file.originalname} must be smaller than ${formatBytes(maxFileSize)}`,
-      }, false);
-    }
+  if (file.originalname.match(new RegExp(`\\.(${FILE_UPLOAD.ALLOWED_FORMATS.join('|')})$`))) {
```

Comment:
is it worth having the `new RegExp(`\\.(${FILE_UPLOAD.ALLOWED_FORMATS.join('|')})$`` in a const so its more readable?

Reply:
yep agreed, will change :)